### PR TITLE
Feature Suggest

### PR DESCRIPTION
**Pull request by aweijun**
Randomly chooses 1 show from the filtered list

## Details
- Add SuggestPredicate
- Add SuggestCommand
- Update TrackermonParser

### CODE REVIEW

File: src/main/java/seedu/trackermon/model/show/SuggestPredicate.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package seedu.trackermon.model.show;
+
+import java.util.function.Predicate;
+
+public class SuggestPredicate implements Predicate<Show> {
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/model/show/SuggestPredicate.java:5]: i think we should write comments for JavaDocs?

File: src/main/java/seedu/trackermon/model/show/SuggestPredicate.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    private final Show testShow;
+
+    public SuggestPredicate(Show show) {
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/model/show/SuggestPredicate.java:9]: Here too!

File: src/main/java/seedu/trackermon/logic/commands/SuggestCommand.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+import java.util.List;
+import java.util.Random;
+
+import seedu.trackermon.logic.commands.exceptions.CommandException;
+import seedu.trackermon.model.Model;
+import seedu.trackermon.model.show.Show;
+import seedu.trackermon.model.show.SuggestPredicate;
+
+public class SuggestCommand extends Command {
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/logic/commands/SuggestCommand.java:13]: Missing comments?

**Review by lowkaiwei98 — COMMENTED** [review]
lgtm, think you missed out some comments.

**Review by Ardentsoul — APPROVED** [review]
LGTM!

File: src/main/java/seedu/trackermon/logic/commands/SuggestCommand.java (hunk 3 above)
Comment (aweijun) [reply src/main/java/seedu/trackermon/logic/commands/SuggestCommand.java:16]: Noted!

**Pull request merged**